src-cpp/server42/src/Workers.cpp

```diff
@@ -33,6 +34,16 @@ using namespace std::literals::chrono_literals;
 
 namespace org::openkilda {
 
+    uint64_t get_current_timestamp() {
+        auto now = std::chrono::time_point_cast<std::chrono::nanoseconds>(
+                std::chrono::high_resolution_clock::now());
+        auto duration = now.time_since_epoch();
+        uint64_t seconds = std::chrono::duration_cast<std::chrono::seconds>(duration).count();
+        uint64_t nanoseconds = std::chrono::duration_cast<std::chrono::nanoseconds>(duration - std::chrono::seconds(seconds)).count();
+        uint64_t timestamp = (seconds << 32ul) + (nanoseconds & 0xFFFFFFFF);
```

Comment:
Bit OR i.e. `|` should be cheaper than int64 sum.

Reply:
do not try to be smarter than GCC. =)